web.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
 var url = require('url');
+var nodemailer = require('nodemailer');
+var transporter = nodemailer.createTransport();
 
```

Comment:
I actually had it the way you said at first, but Kamran didn't like having the Myth gmail acc password exposed because the github is/will be public.  This was nodemailers direct transport. Thoughts?